tests/apollo/util/bft.py

```diff
@@ -430,8 +434,10 @@ async def wait_for_view(self, replica_id, expected=None,
 
         In case of a timeout, fails with the provided err_msg
         """
+        wait_for_liveness_after_view_change = True
```

Comment:
Why do we need this boolean? We always want to `wait_for_liveness` after a view change has completed...